using_images/sti_images/php.adoc

```diff
@@ -80,3 +80,9 @@ The following environment variables set their equivalent property value in the o
 * **OPCACHE_MEMORY_CONSUMPTION**
 ** The OPcache shared memory storage size
 ** Default: 16M
+
+You can also override the entire directory used to load the PHP configuration by setting:
+* **PHPRC**
+  * Sets the path to the php.ini file
```

Comment:
If this is meant to be a sub-bullet, the line should start with `**` (see above lists).